one_fm/one_fm/custom/job_applicant.json

```diff
@@ -1,5 +1,137 @@
 {
  "custom_fields": [
+  {
+   "_assign": null,
+   "_comments": null,
+   "_liked_by": null,
+   "_user_tags": null,
+   "allow_in_quick_entry": 0,
+   "allow_on_submit": 0,
+   "bold": 0,
+   "collapsible": 0,
+   "collapsible_depends_on": null,
+   "columns": 0,
+   "creation": "2024-12-03 13:14:45.325999",
+   "default": null,
+   "depends_on": null,
+   "description": null,
+   "docstatus": 0,
+   "dt": "Job Applicant",
+   "fetch_from": null,
+   "fetch_if_empty": 0,
+   "fieldname": "custom_interview_feedback_rating",
```

Comment:
What is the use of this field, we can update the value to `Applicant Rating` field in Job Applicant

Reply:
@pjamsheer  i will update this to use the applicant rating